src/main/kotlin/blackJack/domain/Dealer.kt

```diff
@@ -0,0 +1,19 @@
+package blackJack.domain
+
+class Dealer : Person("딜러") {
+    private val deck = Deck()
+
+    fun shuffleDeck() {
+        deck.shuffle { it.shuffled() }
+    }
+
+    fun giveCard(person: Person) {
+        person.addCard(deck.getCard())
+    }
+
+    fun overTotalScore16(totalScore: Int = getTotalScore()): Boolean = totalScore >= MINIMUM
```

Comment:
개발자는 코드를 작성하면서 확장성(scalability)에 대한 고민을 항상 해야 합니다.

```suggestion
    fun canAddCard(totalScore: Int = getTotalScore()): Boolean = totalScore < MINIMUM
```